Add EORI number

PatrickBaus opened the pull request:
This PR adds the [EU EORI](https://taxation-customs.ec.europa.eu/customs-4/customs-procedures-import-and-export/customs-procedures/economic-operators-registration-and-identification-number-eori_en) number to the organizations model.

The EORI number is required for import and export in and out of EU. It has an [ISO 3166](https://de.wikipedia.org/wiki/ISO_3166) country prefix and up to 15 digits. Hence I used a 17 character string. There is no checking done on validity.

It would be great to add this important piece of information to the organizations as it allows to quickly look up the EORI number to compile export documents.

I have updated:
- The model
- Migrations
- The tests to make sure it is initially set to an empty string.

genglert commented on creme/persons/migrations/0037_v2_7__organisation_eori.py:
```diff
@@ -0,0 +1,18 @@
+            name='eori',
+            field=models.CharField(
+                max_length=17, verbose_name='EORI number', blank=True,
+                help_text='Economic Operators Registration and Identification number. Required for customs clearance in the EU.',
```
This help_text is only in the migration file, it should be in the model too.

PatrickBaus replied:
@genglert Thx for the hint. I added the same help text to the model as well.

genglert reviewed (commented):
Could you fold your 2 commits too?

PatrickBaus commented:
@genglert Thanks for considering this PR. Sorry for taking so long to reply. I was too busy lately.

 I squashed all commits into one commit and added your suggestion.

genglert commented:
No problem with the delay, it's OK to be busy & work asynchronously <3
